src/api/auth/test/e2e/signup-flow.test.js

```diff
@@ -2,22 +2,24 @@
 const { createServiceToken, hash } = require('@senecacdot/satellite');
 const fetch = require('node-fetch');
 
-const { cleanupTelescopeUsers, login, logout, USERS_URL } = require('./utils');
+// We need to get the URL to the auth service running in docker, and the list
+// of allowed origins, to compare with assumptions in the tests below.
+const { login, logout, USERS_URL, cleanupTelescopeUsers } = require('./utils');
 
 // The user info we'll use to register. We have the following user in our login
 // SSO already:
 //
 // | Username    | Email                       | Password  | Display Name    |
 // |-------------|-----------------------------|-----------|-----------------|
-// | user2 | user2@example.com | user2pass | Galileo Galilei |
+// | user2       | user2@example.com           | user2pass | Galileo Galilei |
 const galileoGalilei = {
   firstName: 'Galileo',
   lastName: 'Galilei',
   email: 'user2@example.com',
   displayName: 'Galileo Galilei',
   isAdmin: false,
   isFlagged: false,
-  feeds: ['https://imaginary.blog.com/feed/hans'],
+  feeds: ['http://localhost:8888/feed.xml'],
   github: {
     username: 'hlippershey',
     avatarUrl:
```

Comment:
The form schema doesn't accept empty URLs as every GH account has a default avatar. 
`avatarUrl: Yup.string().url().required()`.
Should we allow it just for testing?

Reply:
@humphd

Reply:
It's on the next line down: `'https://avatars.githubusercontent.com/u/33902374?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',`